config/initializers/pusher.rb

```diff
@@ -0,0 +1,7 @@
+require 'pusher'
+Pusher.app_id = ENV["PUSHER_APP_ID"]
+Pusher.key = ENV["PUSHER_KEY"]
+Pusher.secret = ENV["PUSHER_SECRET"]
+Pusher.cluster = ENV["PUSHER_CLUSTER"]
```

Comment:
you will need to add these configurations in another file and ignore that on not application.yml